### PR TITLE
Add suppport for Artix Linux

### PR DESCRIPTION
**Pull request by ghost**
I added a line to `fetcher.sh` so that people who use Artix can have support when running `!fetch`.

### CODE REVIEW

**Comment by eepykate** [comment]
disagree - if a distro decides not to do the thing that every other distro does, people making general scripts shouldn't go out of their way to support them.

**Comment by ghost** [comment]
Yes, but that doesn't mean support for the distro isn't supported.

**Comment by eepykate** [comment]
Artix is open-source, right? Why not contribute there instead? It'll make a lot of stuff easier for lots of other projects.

File: fetcher.sh [hunk 1]
```diff
@@ -24,8 +24,14 @@ EOF
...
-	. "/etc/os-release"
-
+	if [[ -f "/etc/os-release" ]] then
+		. "/etc/os-release"
+	fi
+	# Artix support --- On artix
+	# /etc/os-release doesnt exist
+	if [[ ! -f "/etc/os-release" ]] then
+		. "/etc/artix-release"
+	fi
```
Comment (eepykate) [inline fetcher.sh:34]: For starters, this is a shell project, you're using bash code.

second, you replace something that's done in 1 line with something that's slower *and* done in 3 lines.

third, 5 lines to support a crappy distro that nobody uses?

If I were making this change, which again, I am 100% against, I would do

```
. /etc/os-release 2>/dev/null
. /etc/artix-release 2>/dev/null
```

**Comment by aosync** [comment]
I agree Artix Linux is of poor design, but IMO if we know of this issue, we can fix this in the meantime. I don't think the goal of that script is to be the smallest, but to just work.

**Comment by DRG-developer** [comment]
Maybe print that /etc/os-release isn't found, and that you should consult your OS handbook, so that we don't have to create a new patch for each and every “special” OS? or tell em to fill it in for themselves?

**Comment by eepykate** [comment]
Implemented in #69, among other stuff.

**Pull request closed without merging**